Source/Scene/Cesium3DTileset.js

```diff
@@ -3,7 +3,7 @@ define([
         '../Core/Cartesian3',
         '../Core/Cartographic',
         '../Core/Check',
-        '../Core/ClippingPlaneCollection',
+        '../Scene/ClippingPlaneCollection',
```

Comment:
Now that `ClippingPlaneCollection` is in a different folder make sure it's ordered in the correct location in this list as well. Fix throughout.